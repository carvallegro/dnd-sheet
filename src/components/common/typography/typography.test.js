import React from 'react'
import { shallow } from 'enzyme'

import PageTitle from './index'

describe('<PageTitle />', () => {
  it('renders', () => {
    const wrapper = shallow(<PageTitle>Test</PageTitle>)
    expect(wrapper).toMatchSnapshot()
  })
})
