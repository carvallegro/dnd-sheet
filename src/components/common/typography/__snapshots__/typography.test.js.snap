// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> renders 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  ",
          "
    font-family: Lora;
    font-size: 28pt;
    font-weight: bold;
    color: ",
          [Function],
          ";
  ",
          ";
  grid-area: ",
          [Function],
          "
  margin: 0;
",
        ],
      },
      "displayName": "styled.h1",
      "foldedComponentIds": Array [],
      "propStyle": Object {
        "gridArea": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h1",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Test
</StyledComponent>
`;
